packages/flutter/test/rendering/paragraph_test.dart

```diff
@@ -1064,8 +1064,13 @@ void main() {
         ),
       );
       selection = paragraph.selections[0];
-      expect(selection.start, 0); // [how ]are you
-      expect(selection.end, 4);
+      if (isBrowser && !isCanvasKit) {
```

Comment:
@chunhtai I think we should file a bug and skip this test on the HTML renderer instead?

Reply:
yeah I think it make more sense to skip it. but do you know why it was passing for html render before?